how-to-enable-trace-listener-for-asp-net-core-projects.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Problems in ASP.NET Core applications
+description: How to enable Trace Listener for ASP.NET Core applications when crashing
+type: how-to
+page_title: How to enable Trace Listener for ASP.NET Core projects 
+slug: how-to-enable-trace-listener-for-asp-net-core-projects 
+position: 
+tags: ASP.NET Core, Configuration
+ticketid: 1383767
+res_type: kb
+---
+
+## Environment
+<table>
+	<tr>
+		<td>Product</td>
+		<td>Progress® Telerik® Reporting</td>
+	</tr>
+	<tr>
+		<td>Project Type</td>
+		<td>ASP.NET Core</td>
+	</tr>
+	<tr>
+		<td>.Net Framework</td>
+		<td>.NET Core 2.1+</td>
+	</tr>
+</table>
+
+
+## Description
+When the application crashes while working with Telerik Reporting, the way to determine what has caused the problem is described in the following steps.
```

Comment:
Problems in the application may or may not cause a crash. Other issues can also be diagnosed using trace/fiddler/etc.